.vib/reportserver/goss/vars.yaml

```diff
@@ -0,0 +1,21 @@
+binaries:
+  - gosu
+  - java
+  - mysql
+  - render-template
+files:
+  - mode: "0664"
+    paths:
+      - /opt/bitnami/reportserver/WEB-INF/classes/persistence.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/reportserver.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/rsinit.properties
+      - /opt/bitnami/reportserver/WEB-INF/web.xml
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/tomcat/webapps
+      - /opt/bitnami/reportserver
+      - /opt/bitnami/reportserver/WEB-INF/classes
+      - /opt/bitnami/reportserver/WEB-INF/deploy
+      - /opt/bitnami/reportserver/WEB-INF/lib
```

Comment:
I haven't found any mention to the `.../WEB-INF/deploy` and `.../WEB-INF/lib` folders in our logic that would require checking they exist and their permissions, could you point me to the relevant code?

Besides that, I understand we are checking the `.../WEB-INF/classes` dir and its permissions because is the configDir used by Tomcat, right? The issue again here is that we are not really setting permissions nor creating this specific directory.